flutter/shell/platform/tizen/flutter_tizen_view.cc

```diff
@@ -145,6 +145,12 @@ void FlutterTizenView::Resize(int32_t width, int32_t height) {
   tizen_view_->SetGeometry(geometry);
 }
 
+void FlutterTizenView::ExitApplication() {
+  if (initialization_complete_) {
+    ui_app_exit();
+  }
```

Comment:
Could you explain a bit more? According to the Linux implementation (`system_exit_application`),

- If `kExitTypeRequired` or **not** `app_initialization_complete`, `quit_application` immediately.
- Otherwise, send a cancelable `request_app_exit` request. In the response callback, if `kExitResponseExit`, run `quit_application`.

Your implementation doesn't seem to match the behavior. I'd recommend moving the `initialization_complete_` field to the `PlatformChannel` class.

Reply:
I referred to window implementation, but I misunderstood it.
Moved codes to PlatformChannel and modified its behavior.
Please review again. Thanks